Sprout/Models/Task.h

```diff
@@ -0,0 +1,23 @@
+//
+//  Task.h
+//  Sprout
+//
+//  Created by Sarah Wang on 7/13/21.
+//
+
+#import <Parse/Parse.h>
+#import <Foundation/Foundation.h>
+#import "Goal.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Task : PFObject<PFSubclassing>
+
+@property (nonatomic, strong) NSDate *createdAt;
+@property (nonatomic, strong) NSDate *expiresAt;
```

Comment:
I think an NSTimeInterval makes more sense here. `duration` ? Then if you have a creationTime and a duration you can calculate the expiration. 

Also, duration is more flexible because maybe you don't want to expire until it's a certain amount of time after it's started.